MSAL/test/unit/native_auth/public/MSALNativeAuthUserAccountResultTests.swift

```diff
@@ -92,16 +94,102 @@ class MSALNativeAuthUserAccountResultTests: XCTestCase {
         let rawIdToken = "rawIdToken"
 
         cacheAccessorMock = MSALNativeAuthCacheAccessorMock()
+        silentTokenProviderFactoryMock = MSALNativeAuthSilentTokenProviderFactoryMock()
 
         sut = MSALNativeAuthUserAccountResult(
             account: account!,
             rawIdToken: rawIdToken,
             configuration: MSALNativeAuthConfigStubs.configuration,
-            cacheAccessor: cacheAccessorMock
+            cacheAccessor: cacheAccessorMock,
+            silentTokenProviderFactory: silentTokenProviderFactoryMock
         )
         try super.setUpWithError()
     }
 
+    // MARK: - get access token tests
+
+    func test_getAcessToken_succesfullyReturnsAccessToken() async {
```

Comment:
nit: typo "getAcessToken" and "succesfully"

Same on the tests in 144 and 179

Reply:
Fixed